src/tls.c

```diff
@@ -388,21 +392,28 @@ int BuildTlsFinished(WOLFSSL* ssl, Hashes* hashes, const byte* sender)
 {
     int         ret;
     const byte* side;
-    byte        handshake_hash[HSHASH_SZ];
+    byte*       handshake_hash;
```

Comment:
Its not too big. The async calls for QuickAssist benefit from using allocation buffers here because they can be used directly instead of having to do a malloc/memcpy.